chart/Chart.yaml

```diff
@@ -20,7 +20,18 @@
 apiVersion: v1
 name: airflow
 version: 1.0.0
-description: Helm chart to deploy Apache Airflow
+appVersion: 2.0.2
+description: Helm chart to deploy Apache Airflow, A platform to
```

Comment:
```suggestion
description: Helm chart to deploy Apache Airflow, a platform to
```

Reply:
Done in https://github.com/apache/airflow/pull/15866/commits/8df82b44b89c252f66883d9cadadb915a9e34423